insanely_fast_whisper_rocm/cli/commands.py

```
@@ -391,13 +398,48 @@ def _register_signal(sig: int) -> None:
+                    max_speakers=max_speakers,
+                    device=diarization_device,
+                    hf_token=constants.HF_TOKEN,
+                )
+                reporter.on_postprocess_finished("diarization")
+            except DiarizationError as exc:
+                if not quiet:
+                    click.secho(
+                        f"\u26a0\ufe0f  Diarization failed: {exc}",
+                        fg="yellow",
+                    )
+            except Exception as exc:  # pragma: no cover — defensive
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)